include/rtl_433_devices.h

```diff
@@ -12,7 +12,6 @@
 		DECL(elv_em1000) \
 		DECL(elv_ws2000) \
 		DECL(lacrossetx) \
-		DECL(acurite5n1) \
```

Comment:
@rct - with respect to protocol numbering, weewx-sdr does not care.  it parses the rtl_433 output (json or non-json), so protocol numbering does not affect how it behaves.